WebDriverAgentLib/FBAlert.m

```diff
@@ -123,7 +123,14 @@ - (BOOL)acceptWithError:(NSError **)error
       withDescriptionFormat:@"Failed to find accept button for alert: %@", alertElement]
      buildError:error];
   }
-  return [defaultButton fb_tapWithError:error];
+  [defaultButton tap];
```

Comment:
1. Yes, since I don't see any other possibilities. Also, the code will tap a button for the second time only if it is still visible after the first tap and if the screen orientation is different from portrait, which makes this less likely to happen.
Maybe you have any idea on how to distinguish system security alert from a custom alert? 

2. The test needs to be executed on iPad only and it is necessary to invoke system security alert, which requires changes in our test integration app design. Would it be easy to implement it with the current stuff? 

The easiest solution is just to keep it as is and say people to change to landscape if they need to handle system alerts in landscape. It's anyway XCTest bug }:-@